client/src/components/PicCarousel.css

```diff
@@ -31,10 +31,18 @@
 }
 
 .carousel-caption a {
-	background-color: #faea00;
+	background-color: #eed202;
 	color: black;
 	border-radius: 0;
 	padding: 10px 30px 10px 30px;
 	box-shadow: -2px 2px 4px rgba(0, 0, 0, 0.42);
 	border: 0;
 }
+
+span.carousel-control-next-icon {
+	background-image: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' fill='%23eed202' width='8' height='8' viewBox='0 0 8 8'%3e%3cpath d='M2.75 0l-1.5 1.5L3.75 4l-2.5 2.5L2.75 8l4-4-4-4z'/%3e%3c/svg%3e");
```

Comment:
Thuis can work, is it not possible to pass in the icon itself, it is not a show stopper but if possible and if you have capacity you can consider that